jerry-core/include/jerryscript-core.h

```diff
@@ -100,6 +100,7 @@ typedef enum
   JERRY_FEATURE_SYMBOL, /**< symbol support */
   JERRY_FEATURE_DATAVIEW, /**< DataView support */
   JERRY_FEATURE_PROXY, /**< Proxy support */
+  JERRY_FEATURE_GLOBAL_THIS, /**<GlobalThisValue support */
```

Comment:
This must be after `JERRY_FEATURE_REALM`. Also add a pace after `/**<`

Reply:
OK, done